src/Loader.php

```diff
@@ -294,61 +294,63 @@ public static function register_scripts() {
 			return;
 		}
 
+		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
```

Comment:
Turning `SCRIPT_DEBUG` to false resulted in missing files.  Do I need to do something else to get these minified files built?

<img width="567" alt="Screen Shot 2020-04-28 at 2 18 09 PM" src="https://user-images.githubusercontent.com/10561050/80482234-83d51d80-895c-11ea-9add-9ce5e39ca3af.png">

Reply:
I am unable to reproduce this - are you on commit 5a3ef99f93e700cc5a0e5aa0ab87b9ef44a50ac6?

Can you provide more details as to how you were building the app? `npm start` vs. `npm run build`, etc etc.

Reply:
This was under `npm start`.  I thought modifying `SCRIPT_DEBUG` was all that was needed to get use minified scripts.  If `npm run build` is required to get this working, then this is working as expected 👍

Reply:
That's very odd. I just ran `rm -rf dist/` and `npm start` and I see both minified and unminified files in the `dist/` folder.

Reply:
Restarted a couple times and the JS files were oddly built in the root wc-admin folder.  But I deleted my local branch and pulled the latest changes and so far `npm start` seems to work as well for minified files.

I am getting one console error related to the homepage now:

<img width="466" alt="Screen Shot 2020-04-28 at 8 36 40 PM" src="https://user-images.githubusercontent.com/10561050/80519181-412d3880-8990-11ea-8fe1-c77eb511052f.png">

Reply:
I'm looking into this and will comment if I figure out the resolution, but... paging @psealock - this error goes away if I comment these lines out: https://github.com/woocommerce/woocommerce-admin/blob/717be1ab5183a7d4d0452a6f78a9917e9a6ae7ae/client/profile-wizard/index.js#L244-L250

Reply:
Oh and to fill in some missing context - this problem was introduced by lazy loading the `ProfileWizard` component. If we remove the `lazy` and `Suspense` .. it works fine.

Reply:
This is a really odd one because the Warning about the render function not being pure is wrong. If you replace the output of ProfileWizard to be a string, the error occurs still.

If we go back to hydrating these components outside of the `compose` with `withSelect` and `withDispatch` etc, then the error goes away. For example:

```js
const HydradedProfileWizard = withSettingsHydration( 'general', {
	general: window.wcSettings.preloadSettings.general,
} )( ProfileWizard );
```
I'm not sure why that would be.

Reply:
I went ahead and pushed a fix in https://github.com/woocommerce/woocommerce-admin/pull/4094/commits/46572ed05bf34df2e2a0ca00a3fb602c10f840b4. Feel free to revert if there is a better way. If we do stick with this, we should also update other instances of this pattern of lumping `with*Hydration` into the `compose` for wp.data functions.

Reply:
Also want to note here that this expression results in an error when the ternary is false:

```js
hydrateSettings
? withSettingsHydration( 'general', {
		general: window.wcSettings.preloadSettings.general,
	} )
: () => ( {} ),
```
I think `() => ( {} )` should be something more like `() => Comp => ( <Comp /> )`;

Reply:
> I think () => ( {} ) should be something more like () => Comp => ( <Comp /> );

Yep, you're right - I'll change to use lodash's `identity()`.